### PR TITLE
Adapt chat header and datepicker

### LINKED ISSUE #173: The colors over the application should be coherent (using the colors in the theme.scss)
especially an issue in the chat

### PR DESCRIPTION
**Pull request by daveking7**
closes #173 
Adapt: DatePicker, Header of Chat, Don't rerender ChatItems every second, Hand.

### CODE REVIEW

File: src/components/views/Chat.js [hunk 1]
```diff
@@ -144,13 +146,15 @@ const Chat = () => {
...
 
-                // append new messages at the bottom
-                setMessages((prev => [...prev, ...receivedMessages]))
+
 
                 if (receivedMessages.length){
                     console.debug("successfully fetched " + receivedMessages.length + " message" + (receivedMessages.length === 1 ? "" : "s"))
+                    // append new messages at the bottom
+                    setMessages((prev => [...prev, ...receivedMessages]))
+                    setNumOfMessages(prev => prev + receivedMessages.length)
```
Comment (daveking7) [inline src/components/views/Chat.js:155]: only update if new messages arrived

File: src/styles/views/UserHand.scss [hunk 2]
```diff
@@ -33,6 +33,7 @@
   }
 
   &.card-list {
+    margin-top: 0;
```
Comment (daveking7) [inline src/styles/views/UserHand.scss:36]: @serainaschraff This fixes that 
image

Comment (serainaschraff) [reply src/styles/views/UserHand.scss:36]: great :)

**Review by serainaschraff — APPROVED** [review]
I am not sure if it has been adressed here but on my desktop it looks like this at the bottom uf the screen in the chat view:

grafik


Datepicker looks a lot better :)

**Pull request merged**